components/datatypes/industry-verticals/telecom-subscription.schema.json

```diff
@@ -0,0 +1,211 @@
+          "type": "string",
+          "description": "The human readable name for the subscription."
+        },
+        "xdm:type": {
+          "title": "Type",
+          "type": "string",
+          "description": "The scope of entitlement in relation to how many people are covered by the subscription."
+        },
+        "xdm:country": {
+          "title": "Country",
+          "type": "string",
+          "description": "The country that the subscription contractual and agreement terms are rooted in."
+        },
+        "xdm:startDate": {
+          "title": "Start date",
+          "type": "string",
+          "format": "date",
+          "description": "The date the subscription begins."
+        },
+        "xdm:endDate": {
+          "title": "End date",
+          "type": "string",
+          "format": "date",
+          "description": "The date the current subscription term ends."
+        },
+        "xdm:term": {
+          "title": "Term",
+          "type": "integer",
+          "description": "The numeric value of the term."
+        },
+        "xdm:termUnitOfTime": {
+          "title": "Term unit of time",
+          "type": "string",
+          "description": "The unit of time for the term period."
+        },
+        "xdm:renew": {
+          "title": "Renew",
+          "type": "string",
+          "description": "The agreed way that the subscription may continue after the end date."
+        },
+        "xdm:topUp": {
+          "title": "Top up",
+          "type": "string",
+          "description": "Agreed terms for how consumable aspects of a subscription are repurchased during a billing period."
+        },
+        "xdm:status": {
+          "title": "Status",
+          "type": "string",
+          "description": "The current status of the subscription."
+        },
+        "xdm:contractID": {
+          "title": "Contract ID",
+          "type": "string",
+          "description": "Unique ID for the contract that governs this subscription."
+        },
+        "xdm:paymentMethod": {
+          "title": "Payment method",
+          "type": "string",
+          "description": "The payment method for recurring payments."
+        },
+        "xdm:billingPeriod": {
+          "title": "Billing period",
+          "type": "string",
+          "description": "The duration between billings."
+        },
+        "xdm:billingStartDate": {
+          "title": "Billing start date",
+          "type": "string",
+          "format": "date",
+          "description": "The date when the first bill is due."
+        },
+        "xdm:chargeMethod": {
+          "title": "Charge method",
+          "type": "string",
+          "description": "The way the billing is setup to charge the customer."
+        },
+        "xdm:paymentStatus": {
+          "title": "Payment status",
+          "type": "string",
+          "description": "The standing of the account."
+        },
+        "xdm:category": {
```

Comment:
I see a lot of properties being added to the subscription data type. Since they are product properties and already defined in Product class/mixins. Is there a way we use them here. ?

Example:
category
subcategory
SKU